src/wallet/wallet.h

```diff
@@ -725,13 +725,22 @@ class CWallet final : public WalletStorage, public interfaces::Chain::Notificati
 
     bool CreateTransactionInternal(const std::vector<CRecipient>& vecSend, CTransactionRef& tx, CAmount& nFeeRet, int& nChangePosInOut, bilingual_str& error, const CCoinControl& coin_control, bool sign);
 
+    /** Wallet was initially synced with the mempool. */
+    bool m_mempool_synced{false};
```

Comment:
d37756f380b27d259a7818078ddf02caa0a4578a

Make it atomic? Currently accessed in multiple threads.

Reply:
no way